### ISSUE TITLE
No types are exported for UseFormRegister, UseFormWatch, FieldValues, etc.

### ISSUE THREAD

**Issue by aditya-databrain** [vue-composable-form/vue-composable-form#6]
Please export the types also as I am getting so many type errors and explicitly creating types sometimes has linting errors.
Thank you!

**Comment by kareemjeiroudi** [comment]
True, also I had to patch the package in order for VS Code to infer the types.

@SnowingFox I think you only have to add 


```json
{
.
.
.
"exports": {
    ".": {
      ....
      "types": "./dist/index.d.ts"
    }
  }
}
```

to `package.json`

### Update

I checked out the package, it turns out the types are exported. so my suggested solution is relevant 
https://github.com/vue-use-form/vue-use-form/blob/123731014c1c18be214c1de65bb208b1dbcd99d0/packages/core/package.json#L26
